pkg/aws/awscontrolplane/mutate_awscontrolplane.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/giantswarm/aws-admission-controller/v2/pkg/mutator"
 )
 
+const defaultnamespace = "default"
+
```

Comment:
I just added this to make the linter happy since it complained due to duplicated code